src/Components/Join-now.js

```diff
@@ -90,6 +91,27 @@ export default function JoinNow() {
     }
   }
 
+  const getStepperItemClass = (currentStep, stepIndex) => {
```

Comment:
Are you using this anywhere? Looks like you made a class and aren't using this at all.